script/download-pr

```diff
@@ -4,36 +4,32 @@ import json
 import os
 import sys
 
-closed_pr = set( [1197 , 1201, 1235, 1244, 1245, 1252, 1261, 1262, 1366,
-                  1357, 1282, 1279, 1267, 1273, 1305, 1312, 1313, 1339 , 1346,
-                  1354, 1370,1371, 1381])
+url_fmt = "https://api.github.com/repos/Statoil/libres/pulls/%d"
 
-def getPRList( api_token, first_pr ):
-    pr = first_pr
+def getPRList( api_token, pr1, pr2):
+    pr = pr1
     pr_list = []
     prev_len = 0
     sys.stderr.write("Downloading PR: ")
     while True:
-        if pr in closed_pr:
-            pr += 1
-        else:
-            url = "https://api.github.com/repos/Ensembles/ert/pulls/%d" % pr
-            response = requests.get( url , {"access_token" : api_token})
-            for i in range(prev_len):
-                sys.stdout.write("\b")
-
-            sys.stdout.write("%d" % pr )
-            sys.stdout.flush()
-            prev_len = len("%s" % pr)
-            
-            if response.status_code == 200:
-                pr_list.append( json.loads( response.content ) )
-                pr += 1
-            else:
-                break
-
-
-        
+        url = url_fmt % pr
+        response = requests.get( url , {"access_token" : api_token})
+        for i in range(prev_len):
+            sys.stdout.write("\b")
+
+        sys.stdout.write("%d" % pr )
+        sys.stdout.flush()
+        prev_len = len("%s" % pr)
+
+        if response.status_code == 200:
+            pr_list.append( json.loads( response.content ) )
+        pr += 1
+
+        if pr > pr2:
```

Comment:
I think `for pr in range(pr1, pr2+1):` instead of the `while True:` and `break` construction would be nicer..

Reply:
I agree; this script is Copy & Pasted to three repositories. Will leave it as is now.